gradle/template.gradle.properties

```diff
@@ -0,0 +1,43 @@
+# See gradlew :helpLocalSettings for more information on configuring this file.
```

Comment:
Not sure this comment is still valid.

Reply:
That command will still give the help output from https://github.com/apache/solr/blob/main/help/localSettings.txt which describes the concepts, rather than the original gradle task.

It could be moved into the template file as a comment instead though

Reply:
Yea my comment was about `configuring this file` - agree the concepts are good. I'm not saying we need to change this completely, but it would be good to consolidate to avoid having to go look another place.

Reply:
I've moved the help content into the template so it's all self contained